plugins/bernard.md

```diff
@@ -37,16 +40,30 @@ On the other side of the message queue you must set up a consumer:
 
 ~~~ php
 use Bernard\Consumer;
-use League\Tactician\Bernard\Router;
+use Bernard\Router\SimpleRouter;
+use League\Tactician\Bernard\Receiver\SingleBusReceiver;
 use League\Tactician\CommandBus;
 use Symfony\Component\EventDispatcher\EventDispatcher;
 
 // inject some middlewares
 $commandBus = new CommandBus([]);
 
-$router = new Router($commandBus);
+$receiver = new SingleBusReceiver($commandBus);
+
+$router = new SimpleRouter();
+
+$router->add('League\Tactician\Command', $receiver);
```

Comment:
Maybe this code sample could use a couple comment headers to help break it up? Something like: 

`
// create the command bus that receives the queued commands
$commandBus = new CommandBus([/* middlewares go here */]);

// Wire the command bus into Bernard's routing system
$receiver = new SingleBusReceiver($commandBus);
$router = new SimpleRouter();
$router->add('League\Tactician\Command', $receiver);

// Finally, create the Bernard consumer that runs through the pending queue
$consumer = new Consumer($router, new EventDispatcher());
$consumer->consume($queue);
`